image:
    using_rgb: true
    using_depth: false
